.github/workflows/metrics.yaml

```diff
@@ -0,0 +1,21 @@
+name: Scheduled Metrics
```

Comment:
This should land in the `openshift-helm-charts/charts` right? Why should we ignore this in the release?

Reply:
It can run either in development or charts but not both. If its charts only it should not be in this PR, if it is development only it needs to be in release_info.json. Alternatively add logic so it only runs if it is in the correct repo.

Reply:
This should be prod only, it wouldn't make sense to collect data in the development. But code can be in both repos, we just need to turn off the corn job in the repo settings.